rust-src/id/benches/aggr_dlog_commit_point.rs

```diff
@@ -0,0 +1,25 @@
+use criterion::*;
+use curve_arithmetic::*;
+use id::sigma_protocols::{aggregate_dlog::*, common::*};
+use pairing::bls12_381::G1;
+use rand::*;
+
+fn bench_aggr_dlog_commit_point(c: &mut Criterion) {
```

Comment:
Please provide a brief comment as to the purpose of this benchmark.

Reply:
Added minimal description. I guess there is no real purpose, as we found out it isn't used. If we need it in the future we now know that it is faster.